nntrainer/src/conv2d_layer.cpp

```diff
@@ -159,6 +186,31 @@ sharedConstTensor Conv2DLayer::backwarding(sharedConstTensor derivative,
              input_dim.width() + padding[1] * 2);
   ret.setZero();
 
+  // Calculate DelK
+  //
+  //                          derivative
+  //                        +------|------+
+  //                        |------|------|
+  //  [filter_size (height) |------|------|
+  // (=derivative->channel) |------|------|
+  //                        +------|------+
+  //                     [derivative->height
+  //                       * derivative->width (width)]
+  //
```

Comment:
Thanks for the description in person.
can you please add a 1 line comment here like this is the 2D derivative matrix with height ... and width ...
Also, change to multi line comment format `/** */`